src/renderer/components/+workloads-pods/pods.tsx

```diff
@@ -82,7 +82,7 @@ export class Pods extends React.Component {
           sortingCallbacks={{
             [columnId.name]: pod => getConvertedParts(pod.getName()),
             [columnId.namespace]: pod => pod.getNs(),
-            [columnId.containers]: pod => pod.getContainers().length,
+            [columnId.containers]: pod => pod.getContainerStatuses().length,
```

Comment:
I find it more intuitive to keep container amount as the sorting criteria. Sorting by the statuses is not clear enough.

Reply:
Well the bug was that you found the previous sorting mechanism unintuitive. With this the sorting matches the visuals.